packages/http-client/src/remote-admin-api.ts

```diff
@@ -86,9 +106,23 @@ export class RemoteAdminApi implements AdminApi {
         )}`,
       },
     })
-    return response.models.map((modelStreamIDString: string) => {
-      return StreamID.fromString(modelStreamIDString)
+    return response.models.map((id) => StreamID.fromString(id))
+  }
+
+  async getIndexedModelsWithFieldIndices(): Promise<Array<ModelFieldsIndex>> {
```

Comment:
do we need a new method?  Could we just add an optional `args` object to the existing `getIndexedModels` function?

Reply:
This would change the return type from `StreamID` to `ModelFieldsIndex`. If we're good with making this a breaking change in the API, we could simplify a bit of the code in this PR.